scripts/zones/Uleguerand_Range/IDs.lua

```diff
@@ -17,6 +17,10 @@ zones[dsp.zone.ULEGUERAND_RANGE] =
         KEYITEM_OBTAINED        = 6404, -- Obtained key item: <keyitem>.
         NOTHING_OUT_OF_ORDINARY = 6415, -- There is nothing out of the ordinary here.
         CONQUEST_BASE           = 7062, -- Tallying conquest results...
+        SOMETHING_GLITTERING    = 7333, -- You see something glittering beneath the surface of the ice.
+                                        -- There are many cold Moblin Hotroks scattered around the area.
+        WHAT_LIES_BENEATH       = 7334, -- Could someone be trying to melt the ice to retrieve what lies beneath?
```

Comment:
```
SOMETHING_GLITTERING    = 7333, -- You see something glittering beneath the surface of the ice.
WHAT_LIES_BENEATH       = 7334, -- There are many cold <item> scattered around the area. Could someone be trying to melt the ice to retrieve what lies beneath?
```

Gotta keep the right text (and text replacements) in these IDs, so they'll be updated correctly if they shift.